README.md

```diff
@@ -98,14 +100,6 @@ less azuredeploy.parameters.json
 
 # connect to the coordinator
 ./connect.sh
-
-# ALTERNATIVATELY instead of ./connect.sh you can do the following
-
-# Delete security rule 103 to be able to connect
-./delete-security-rule.sh
-
-# When your cluster is ready, it will prompt you with the connection string, connect to coordinator node
-ssh -A pguser@<public ip of coordinator>
```

Comment:
instead of removing this line, maybe we should provide port 3456 too

Reply:
Do you think it is valuable? `./connect.sh` seems enough. There wasn't any connect script before and there was only this alternative in the past, so now using `connect.sh` is what we do

Reply:
true, it doesn't seem valuable